tensorflow/core/kernels/cwise_op_gpu_arg.cu.cc

```diff
@@ -13,14 +13,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-#if GOOGLE_CUDA
+#if GOOGLE_CUDA || TENSORFLOW_USE_ROCM
 
 #include "tensorflow/core/kernels/cwise_ops_gpu_common.cu.h"
 
 namespace tensorflow {
 namespace functor {
+#if GOOGLE_CUDA
```

Comment:
guessing this is just a cosmetic change then?

Reply:
In a sense. It says, "we support the op in principle, but we don't implement it for complex64 or complex128 at this time."